CONTRIBUTING.md

```diff
@@ -1,14 +1,36 @@
 # Contributing
 
-## General Workflow
+## Branching Strategy
+
+Feature -> Develop -> Master -> Staging -> Deployment
+
+- Deployment: actual production environment available publically
```

Comment:
publicly?

Reply:
ahh yes, that's a typo. I will fix that - thanks @leungleoqin